pages/docs/advanced.js

```diff
@@ -11,7 +11,7 @@ import TaggedTemplateLiterals from '../../sections/advanced/tagged-template-lite
 import ServerSideRendering from '../../sections/advanced/server-side-rendering'
 
 const Advanced = () =>
-  <DocsLayout title="Advanced">
+  <DocsLayout title="Advanced" description="Advanced usage of styled components - Theming, refs, Security, Existing CSS, Media Templates, Tagged Template Literals, SSR">
```

Comment:
The default spellingzzz of styled-components is lowercase and with a dash: "styled-components";

So sorry to be bothering you with this :smile: But we've previously increased our consistency with it

:x: styled components
:x: Styled Components
:x: StyledComponents
:x: / ✅  StyledComponent (Note: Ok to use to refer to the class but not the library)